src/libraries/Microsoft.Extensions.Logging.Console/src/JsonConsoleFormatter.cs

```diff
@@ -0,0 +1,150 @@
+            string message = logEntry.Formatter(logEntry.State, logEntry.Exception);
+            if (logEntry.Exception == null && message == null)
+            {
+                return;
+            }
+            LogLevel logLevel = logEntry.LogLevel;
+            string category = logEntry.Category;
+            int eventId = logEntry.EventId.Id;
+            Exception exception = logEntry.Exception;
+            const int DefaultBufferSize = 1024;
+            var output = new PooledByteBufferWriter(DefaultBufferSize);
+            using (var writer = new Utf8JsonWriter(output, FormatterOptions.JsonWriterOptions))
+            {
+                writer.WriteStartObject();
+                string timestamp = null;
+                var timestampFormat = FormatterOptions.TimestampFormat;
+                if (timestampFormat != null)
+                {
+                    var dateTime = FormatterOptions.UseUtcTimestamp ? DateTime.UtcNow : DateTime.Now;
+                    timestamp = dateTime.ToString(timestampFormat);
+                }
+                writer.WriteString("Timestamp", timestamp);
+                writer.WriteNumber(nameof(logEntry.EventId), eventId);
+                writer.WriteString(nameof(logEntry.LogLevel), GetLogLevelString(logLevel));
+                writer.WriteString(nameof(logEntry.Category), category);
+                writer.WriteString("Message", message);
+
+                if (exception != null)
+                {
+                    writer.WriteStartObject(nameof(Exception));
+                    writer.WriteString(nameof(exception.Message), exception.Message.ToString());
+                    writer.WriteString("Type", exception.GetType().ToString());
+                    writer.WriteStartArray(nameof(exception.StackTrace));
+                    string stackTrace = exception?.StackTrace;
+                    if (stackTrace != null)
+                    {
+                        foreach (var stackTraceLines in stackTrace?.Split(Environment.NewLine))
+                        {
+                            writer.WriteStringValue(stackTraceLines);
+                        }
+                    }
+                    writer.WriteEndArray();
+                    writer.WriteNumber(nameof(exception.HResult), exception.HResult);
+                    writer.WriteEndObject();
+                }
+
+                if (logEntry.State is IReadOnlyCollection<KeyValuePair<string, object>> stateDictionary)
+                {
+                    foreach (KeyValuePair<string, object> item in stateDictionary)
+                    {
+                        writer.WriteString(item.Key, Convert.ToString(item.Value, CultureInfo.InvariantCulture));
+                    }
+                }
+                GetScopeInformation(writer, scopeProvider);
+                writer.WriteEndObject();
+                writer.Flush();
+            }
+            textWriter.Write(Encoding.UTF8.GetString(output.WrittenMemory.Span));
```

Comment:
Does this API exist on netstandard2.0?

Reply:
Looks not. https://apisof.net/catalog/System.Text.Encoding.GetString(ReadOnlySpan%3CByte%3E)

Reply:
Fixed

Reply:
Nope, she's working on fixing this now.